gridstatus/ercot_api/ercot_api.py

```diff
@@ -38,10 +39,9 @@
 HISTORICAL_DAYS_THRESHOLD = 90
 
 
-# This file is publicly available and contains the full list of endpoints and their parameters # noqa
-ENDPOINTS_MAP_URL = (
-    "https://raw.githubusercontent.com/ercot/api-specs/main/pubapi/pubapi-apim-api.json"  # noqa
-)
+# This file is publicly available and contains the full list of endpoints and their parameters
+# "https://raw.githubusercontent.com/ercot/api-specs/main/pubapi/pubapi-apim-api.json"
+ENDPOINTS_MAP_FILE = "gridstatus/ercot_api/pubapi-apim-api.json"
```

Comment:
This file hasn't been changed in 9 months. 

https://github.com/ercot/api-specs/blob/main/pubapi/pubapi-apim-api.json